stories/components/Tab.stories.mdx

```diff
@@ -0,0 +1,315 @@
+        <i class="icon icon-minus icon-lg" /> Remove
+      </button>
+      </div>
+      </Tab>
+      <template #tab-row-extras>
+        <li class="tablist-controls">
+          <button type="button" class="btn-sm role-link" @click="update($event)">
+            <i class="icon icon-plus icon-lg" /> Add
+          </button>
+        </li>
+      </template>
+    </Tabbed>
+    `,
+  };
+};
+
+
+# Tab
+
+Tabs are used to separate information into logical sections and to quickly navigate between them.
+
+### Description
+- Use tabs when there is a large amount of content that can be separated.
+- Use tabs to make the content accessible without reloading the page or compromising on space.
+- Write short tab labels. Try not to use more than two words for a tab label.
+
+- Secondary tabs never appear without a parent set of primary tabs.
+- The number of secondary tabs inside vertical tabs should be as small as possible – not more than six are recommended.
+<br/>
+<br/>
+
+### Types
+
+<br/>
+
+#### Default/Horizontal Tabs
+- Horizontal tabs are best used with ten or fewer tabs. This maintains an uncluttered UI and reduces cognitive load for users.
+- Write short tab labels. Especially with a large number of tabs and little horizontal screen space.
+- Stick to only one row of tabs. Use tab navigation when the content categories fit in a single row.
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      useHash: false,
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+#### Side/Vertical Tabs
+- Vertical tabs provide better scalability for a large number of tabs (ten and more).
+
+<Canvas>
+   <Story
+    name="Vertical"
+    args={{
+      useHash: false,
+      sideTabs: true,
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+
+#### Nested
+##### General construction
+- Tabs can either be arranged horizontally or vertically. 
+- The current tab defines what content is shown in the tab pane. Tabs can be nested in two levels.
+- Secondary tabs only appear as nested tab.
+
+##### Horizontal tabs with vertical secondary tabs.
+<Canvas>
+   <Story
+    name="Nested horizontal/vertical"
+    args={{
+      useHash: false,
+      flat: true,
+    }}>
+    {TabNestedHorizontal.bind({})}
+  </Story>
+</Canvas>
+
+##### vertical tabs with vertical secondary tabs
+<Canvas>
+  <Story
+    name="Nested vertical/vertical"
+    args={{
+      useHash: false,
+      sideTabs: true
+    }}>
+    {TabNestedVertical.bind({})}
+  </Story>
+</Canvas>
+
+
+
+### Add/Remove Vertical Tab
+<Canvas>
+  <Story
+    name="Add/Remove Vertical Tab"
+    args={{
+      tabs: [
+        {
+          "name": "Tab1",
+          "content": "Tab content 1",
+        },
+        {
+          "name": "Tab2",
+          "content": "Tab content 2",
+        }
+      ],
+      useHash: false,
+      sideTabs: true,
+      showTabsAddRemove: true,
+    }}>
+    {AddRemove.bind({})}
+  </Story>
+</Canvas>
+
+### Add/Remove Default Tab
```

Comment:
Could you please change the markup and remove the logic? This is not the built in add/remove feature, so we cannot list as something ready to use. It has, sorry to say it, no purpose for the style guide.

Let's add anything else in the tab. Keeping the button may also be ok, although it may is totally misleading.

In the future we may add also specification in what and how add it, if not e refactoring.